docs/source/using-discovery/workingwithgpu.rst

```diff
@@ -188,29 +188,33 @@ You can find the compatibility of CUDA and TensorFlow versions at the following
 
 For the latest installation, use the TensorFlow pip package which includes GPU support for CUDA-enabled devices::
 
-  srun --partition=gpu --nodes=1 --pty --gres=gpu:1 --ntasks=1 --mem=4GB --time=01:00:00 /bin/bash
+  srun --partition=gpu --gres=gpu:1 --nodes=1 --ntasks=1 --mem=10GB --time=01:00:00 --pty /bin/bash
   module load anaconda3/2022.05
   module load cuda/11.2
```

Comment:
why not cuda/11.4?

I ran through the exercise while only changing this and cudatoolkit=11.4.2, and winded up at the same end result (warnings and all)

Reply:
@visionjo, cuda 11.2 was what was listed on the TF documentation. Since then, the installation instructions over there have been updated to 11.8. We will incorporate that once we get our CUDA kernels updated to that version next month.